src/components/container/BloodPressureVisualization/BloodPressureVisualization.css

```diff
@@ -0,0 +1,79 @@
+body{
+    background: #FFFFFF;
+}
+
+.mdhui-blood-pressure-visualization-pager {
+    width: 100%;
+    padding-top: 13px;
+    padding-bottom: 13px;
+}
+
+.mdhui-blood-pressure-metrics {
+    padding-top: 10px;
+    padding-bottom: 10px;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+}
+
+.mdhui-blood-pressure-metrics-rows{
+    display: flex;
+    flex-direction: row;
+    background-color: #FFFFFF;
+    height: 50%;
+}
+
+.mdhui-blood-pressure-metrics-block{
+    display: flex;
+    flex-direction: column;
+    padding: 16px;
+    border-radius: 6px;
+    border-color:#EDEDED;
```

Comment:
Would use a border color variable from the globals for this - again so it works in dark mode